doc/Learn/Mvux/reactive-migration.md

```diff
@@ -2,7 +2,7 @@
 uid: Uno.Extensions.Reactive.Upgrading
 ---
 
-# Upgrading Extensions Reactive Version
+# Upgrading MVUX
 
 ## Upgrading to Extensions Reactive V5
```

Comment:
This line should be changed to MVUX as well I believe

Reply:
This one refers to the actual package version, which, in this case, is the Uno.Extensions.Reactive V5.